hikari/api/rest.py

```diff
@@ -2995,16 +3001,12 @@ async def refresh_access_token(
         client: snowflakes.SnowflakeishOr[guilds.PartialApplication],
         client_secret: str,
         refresh_token: str,
-        *,
-        scopes: undefined.UndefinedOr[
-            typing.Sequence[typing.Union[applications.OAuth2Scope, str]]
-        ] = undefined.UNDEFINED,
```

Comment:
This shouldn't be removed. This is a part of the Oauth2 spec and Discord does use this field these days.